crates/swc_ecma_transforms_module/src/amd.rs

```diff
@@ -145,7 +145,7 @@ where
 
         // "use strict";
         if self.config.strict_mode {
-            stmts.push(clone_first_use_strict(n).unwrap_or_else(use_strict));
+            stmts.push(clone_first_use_directive(n).unwrap_or_else(use_strict));
         }
```

Comment:
Maybe I'm not understanding what this is doing, but at first glance, does this mean that if an input file has a directive, it would only get hoisted to the top of the file if `strictMode` is enabled? I'm using strict mode, so it doesn't really affect me, but I imagine this would be a bug that may present itself in the future.

Reply:
I guess I must be mis-understanding. I just finished upgrading and tried it out setting strictMode to false and the directive still makes its way to the top. So you can ignore me...it is working as it should.